esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
+{% autoescape off %}
+{{ msgbdy }}
+{% endautoescape %}
```

Comment:
Also, if we want, we could consider including a per-user unsubscribe link that doesn't require logging in, since making people log in to unsubscribe is really annoying.  It would require a bit more code so perhaps it's better to do that separately.